src/test/resources/results/scriptCommand/testScriptCommandForSchemaEvolution_Create_Table/Department/create-pipe-evolve-schema.sql

```diff
@@ -0,0 +1,11 @@
+USE DATABASE SANDBOX_POC1;
+
+USE SCHEMA EMPLOYEES;
+
+CREATE PIPE IF NOT EXISTS Department_pipe
+	AUTO_INGEST = true
```

Comment:
this is use case of schema evolution where a new table is created. Hence initial pipe is created.